### PR TITLE
Added temp item* return

### PR DESCRIPTION
**Pull request by maanpalle**
# Pull Request

## Description

Tried to fix the pathExpr but it is a bit to complicated so added a temporary item()* so it is at least not wrong

Closes none

## Changes

1. made it return an item()*

## Deletes Source Branch?

No:     Work is (partially) done, but more work will be done on the source branch.

## How Many Approvals?

* 1 Approval (Small bugs/hot-fixes)

### CODE REVIEW

File: src/typeInference/annotatePathExpr.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ import astHelper, { IAST } from '../parsing/astHelper';
  */
 export function annotatePathExpr(ast: IAST): SequenceType {
 	const seqType = {
-		type: ValueType.NODE,
+		type: ValueType.ITEM,
```
Comment (yuanzexiong) [inline src/typeInference/annotatePathExpr.ts:13]: Cannot resolve? Item()*?

**Review by yuanzexiong — APPROVED** [review]

File: src/typeInference/annotatePathExpr.ts (hunk 1 above)
Comment (maanpalle) [reply src/typeInference/annotatePathExpr.ts:13]: For the moment yes

**Comment by MTimmerije** [comment]
Seems fine for what it is.

**Review by HectorPeeters — APPROVED** [review]
For now this seems fine

**Pull request merged**